pkg/ocm/ocmWrapper.go

```diff
@@ -275,6 +276,23 @@ func (*DefaultOCMInterfaceImpl) GetStsSupportJumpRoleARN(ocmConnection *ocmsdk.C
 	return response.Body().RoleArn(), nil
 }
 
+// GetBackplaneURL returns Backplane URL based on ocm env
+func (*DefaultOCMInterfaceImpl) GetOCMEnvironment() (*cmv1.Environment, error) {
+	// Create the client for the OCM API:
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create OCM connection: %v", err)
```

Comment:
Let's use the standard error format specifier for golang `%w`, applicable for error values, which wraps the error such that it can later be unwrapped with `errors.Unwrap`.

```suggestion
		return nil, fmt.Errorf("failed to create OCM connection: %w", err)
```